mappings/net/minecraft/item/ComplexItem.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS bar net/minecraft/item/ComplexItem
```

Comment:
Unsure about this name. It's an item that can send a packet every tick when it is in a player's inventory.

Reply:
The reasoning for this class name is listed in the initial ticket message. I am open to suggestions for a different name, though. I also agree it is fairly ambiguous. I could not think of an appropriate name to describe the behaviour at the time though, so opted for the Mojang name used in Bedrock.

Data provided by @kashike through Discord:
```
02a08829 T ComplexItem::ComplexItem(std::string const&, int)
02a08829 T ComplexItem::ComplexItem(std::string const&, int)
02a08845 T ComplexItem::~ComplexItem()
02a08841 T ComplexItem::~ComplexItem()
02a08841 T ComplexItem::~ComplexItem()
```

Reply:
Resolved by d06eb13 after discussion in Discord